ably/rest_channel.go

```diff
@@ -53,8 +53,9 @@ func newRESTChannel(name string, client *REST) *RESTChannel {
 
 // Publish publishes a message on the channel.
 func (c *RESTChannel) Publish(ctx context.Context, name string, data interface{}) error {
+	msg := data.(Message)
```

Comment:
You need to add optional type assertion, current one is forced. Instead, you can do
```
msg, ok := data.(Message)
if (ok) {
connectionKey := data.connectionKey
}
```

Reply:
`data` is always not of type `Message`, it can be a string too

Reply:
Thank you @sacOO7 I have addressed your comment.